src/views/app/Conversation.vue

```diff
@@ -403,18 +403,24 @@
         return value;
     };
 
-    const doSendFiles = async (files: []) => {
+    const showError = ref(false);
+    const errorMessage = ref('');
+    const doSendFiles = async (files: File[]) => {
         for (const f of files) {
+            if (hasSpecialCharacters(f.name)) {
+                showError.value = true;
+                errorMessage.value = 'File name cannot have special characters.';
+                return;
+            }
             const success = await sendFile(chat.value.chatId, f);
             if (!success) {
                 showError.value = true;
+                errorMessage.value = 'The file was to big, the maximum supported size is 20MB.';
```

Comment:
So the only reason the file could not be sent is because the maximum supported size is 20mb?

Reply:
no, also if the name has special characters
https://github.com/threefoldtech/twin_aydo/issues/395